lib/matplotlib/backend_bases.py

```diff
@@ -2012,6 +2029,10 @@ def print_figure(self, filename, dpi=None, facecolor=None, edgecolor=None,
             A list of extra artists that will be considered when the
             tight bbox is calculated.
 
+        backend : str, optional
+            When set, forcibly use this non-interactive backend for rendering.
+            This can be either a standard backend name ("agg", "pdf", "pgf",
+            "ps", "svg") or a custom backend name ("module://...").
```

Comment:
This could use a comment as to why we would ever want to do this.  I'm not sure I understand from the PR what the goal is, so I can imagine users trying to dutifully use this keyword and getting themselves into a mess.

Reply:
done